astropy/io/fits/tests/test_image.py

```diff
@@ -1126,6 +1126,11 @@ def test_hdu_creation_with_scalar(self):
             fits.ImageHDU(data=1)
         with pytest.raises(TypeError, match=msg):
             fits.PrimaryHDU(data=1)
+        # Regression test for https://github.com/astropy/astropy/issues/14527
+        with pytest.raises(TypeError, match=msg):
+            fits.ImageHDU(data=np.array(1))
+        with pytest.raises(TypeError, match=msg):
+            fits.PrimaryHDU(data=np.array(1))
```

Comment:
Huh, weird that this `test_image.py` is not being ignored by coverage. @nstarman , is this setting being ignored? You moved it in #13266 .

https://github.com/astropy/astropy/blob/1b41891bf92ae36d8f1ad40d77377958663e4ca2/pyproject.toml#L359

Reply:
For some reason it complains about coverage on test files while the rest of the tests are still running, but as soon as everything finishes it stops complaining. This also happened to my commit last week, but I have no clue why.

Reply:
This is because we have several jobs in the same build that upload coverage reports. They don't all check the same test suite. The most comprehensive one is also the slowest, so when that one is done is when you should have a look. I won't worry about the complains while the test suite is still running. 😸

Reply:
hm. This is what the coverage docs say:

> Coverage.py will read from “pyproject.toml” if TOML support is available, either because you are running on Python 3.11 or later, or because you installed with the toml extra (pip install coverage[toml]). Configuration must be within the [tool.coverage] section, for example, [tool.coverage.run]. 

AFAIK it should be working. I see `coverage[toml]` in `setup[cfg]`.

Reply:
Do you think we need a `**` instead of a `*`? Or maybe a `*/*` since this is one more level down?

Reply:
Yeah, maybe ``**``.

Reply:
codecov was happy when the change was made in #13266, so maybe those files were already not ignored before that and we would need `**`. To be checked with another PR ?